src/pages/ActivityOverviewPage.js

```diff
@@ -22,37 +13,12 @@ export default function ActivityOverviewPage({ activity, friend }) {
   );
 }
 
-const Title = styled.header`
-  background: #f0e7da;
-  padding: 10px;
-  text-align: center;
-  text-transform: uppercase;
-  color: rgba(71, 39, 35, 0.72);
-  position: sticky;
-  top: 0px;
-  z-index: 2;
-  height: 60px;
-  overflow: hidden;
-`;
-
-const Arrowback = styled(NavLink)`
-  position: fixed;
-  top: -5px;
-  left: -2px;
-`;
-
-const HeaderText = styled.h1`
-  margin-left: 70px;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
-`;
-
 const WrapperCard = styled.section`
   color: rgba(71, 39, 35, 0.72);
   margin: 20px;
   font-size: 25px;
   display: grid;
   grid-template-rows: repeat(2, auto);
   gap: 15px;
+  word-break: break-word;
```

Comment:
This is new to me.

Reply:
got it as hint from Lasse, i think he got it from Michi haha😅
Its really nice, when you have a too long word (what will probably never happen, but just in case 😊)